src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -181,9 +189,37 @@ class ErgoWalletActor(settings: ErgoSettings, boxSelector: BoxSelector)
       sender() ! trackedAddresses.toIndexedSeq
   }
 
+  /**
+    * Clear out wallet boxes which do not belong to UTXO set or unconfirmed transaction outputs
+    * //todo: likely temporary method for 3.2.x
+    */
+  private def paranoidClear(): Unit = {
```

Comment:
Please describe why this is necessary, what was the problem.

Reply:
Please check PR description